verticapy/plotting/_plotly/bar.py

```diff
@@ -48,9 +50,15 @@ def draw(self, **style_kwargs,) -> ...:
         """
         Draws a bar chart using the Plotly API.
         """
-        print(self.data)
-        print(self.layout)
-
+        y_data=self.data['y']
+        x_data=self.layout['labels']
+        fig = px.bar(x=x_data, y=y_data)
```

Comment:
no need to create variable for x_data and y_data, use them directly in your plotting function.